### PR TITLE
Updating Integration Testing example and documentation

### PR DESCRIPTION
**Pull request by agbell**
Updating Integration Testing example and documentation.

The documentation has been simplified to focus directly on use of `WITH DOCKER` and to remove other details.

### CODE REVIEW

File: docs/guides/integration.md [hunk 1]
```diff
@@ -96,61 +93,71 @@ Ouput:
 [info] Run completed in 2 seconds, 954 milliseconds.
 [info] Total number of tests run: 1
 ```
+{% sample lang="Service Dependencies" %}
+
+The Docker compose specifies the application's dependencies.  It is useful for local development. It can be started and stopped using `docker-compose up -d` and `docker-compose down`.
```
Comment (vladaionescu) [inline docs/guides/integration.md:98]:
```suggestion
The Docker compose configuration specifies the application's dependencies.  It is useful for local development and can be started and stopped using `docker-compose up -d` and `docker-compose down`.
```

File: docs/guides/integration.md [hunk 2]
```diff
@@ -96,61 +93,71 @@ Ouput:
...
+```
+
+
 {% endmethod %}
 
-## The Earthfile
+### The Basic Earthfile
 
-We start with a simple Earthfile that can build and create a docker image for our app. See [Basic](./basics) guide for more details on that.
+We start with a simple Earthfile that can build and create a docker image for our app. See the [Basic](./basics) guide for more details, as well as examples in many programming languages.
```
Comment (vladaionescu) [inline docs/guides/integration.md:129]: Add link to examples section here?

File: docs/guides/integration.md [hunk 3]
```diff
@@ -192,107 +196,31 @@ docker:
...
-Our example application has one direct dependency, Postgres, and a helper app `adminer` which is a web UI for postgres. It is strictly not necessary for running our integration tests but useful in local development and serves to show how this solution scales to multiple dependencies.
-{% endhint %}
+Since our service has a docker-compose file of dependencies, running integration tests is easy.
 
-## Integration Testing Step 2 - Run your tests
-
-With our docker-compose ready, we can now add an integration test step to our Earthfile. 
-
-Our integration target needs to copy in our source code and our docker file before it starts the tests:
+Our integration target needs to copy in our source code and our docker file and then inside a `WITH DOCKER` start the tests:
```
Comment (vladaionescu) [inline docs/guides/integration.md:205]:
```suggestion
Our integration target needs to copy in our source code and our Dockerfile and then inside a `WITH DOCKER` start the tests:
```

**Review by vladaionescu — APPROVED** [review]
Some minor suggestions

File: docs/guides/integration.md (hunk 2 above)
Comment (agbell) [reply docs/guides/integration.md:129]: There is a link to the Basics guide. Those were the examples I was referring to, in this case.

Comment (vladaionescu) [reply docs/guides/integration.md:129]: Ah gotcha

**Pull request merged**